dali/dfu/dfuutil.cpp

```diff
@@ -480,6 +478,16 @@ class CFileCloner
         StringBuffer dstdir;
         getLFNDirectoryUsingBaseDir(dstdir, dstlfn.get(), spec.defaultBaseDir.get());
         dstfdesc->setDefaultDir(dstdir.str());
+
+        Owned<IStoragePlane> plane = getDataStoragePlane(cluster1, false);
+        if (plane) // I think it should always exist, even in bare-metal.., but guard against it not for now (assumes initializeStorageGroups has been called)
+        {
+            DBGLOG("cloneSubFile: destfilename='%s', plane='%s', dirPerPart=%s", destfilename, cluster1.get(), boolToStr(plane->queryDirPerPart()));
+            if (plane->queryDirPerPart())
+                dstfdesc->setFlags(FileDescriptorFlags::dirperpart);
```

Comment:
else
    setFlags(FileDescriptorFlags::none);

otherwise if copying from a plane that does support it to one that doesn't it will still have it

Reply:
yes, missed that.
@ghalliday - please see new commit.